test/tdigest_test.rb

```diff
@@ -145,4 +154,77 @@ def test_that_it_has_a_version_number
       tdigest.size.must_equal n
     end
   end
+
+  describe '#+' do
+    it 'works with empty tdigests' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.